### DISCUSSION TITLE
SemiCircleProgress feature request

### DISCUSSION THREAD

**Discussion by charmbyte** [mantinedev/mantine#6716] [Feature requests]
It may be useful to be able to control the styling of the Label element within the SemiCircleProgress component. For instance, in my case, I would like to be able to make the label font size smaller. If this is already possible, I don't see it documented anywhere.

**Comment by rtivital** [comment]
You can change component styles with Styles API – https://mantine.dev/styles/styles-api/

**Comment by charmbyte** [comment]
@rtivital Got it, thank you!
